### ISSUE TITLE
Change year 

### ISSUE THREAD

**Issue by johinsDev** [tresko/react-datepicker#35]
I change year programmatically or with a method of useDatepicker?

**Comment by johinsDev** [comment]
I think that with a simple change i can controlled the number of years and for example i would create a select of years:
```js
 function goToPreviousYear(numYears = 1) {
    setActiveMonths(
      getNextActiveMonth(
        activeMonths,
        numberOfMonths,
        -(numYears * 12 - numberOfMonths + 1)
      )
    );
    setFocusedDate(null);
  }

  function goToNextYear(numYears = 1) {
    setActiveMonths(
      getNextActiveMonth(
        activeMonths,
        numberOfMonths,
        numYears * 12 - numberOfMonths + 1
      )
    );
    setFocusedDate(null);
  }
```
WDYT? 🤔

**Comment by tresko** [comment]
Would you create a pull request? It seems fine to me.

**Comment by johinsDev** [comment]
> Would you create a pull request? It seems fine to me.

Yes off course

**Comment by goxr3plus** [comment]
@johinsDev  Waiting for it :)

**Comment by tresko** [comment]
I will do it today or tomorrow.

**Comment by tresko** [comment]
Included in v2.2.
